libmscore/read206.cpp

```diff
@@ -3001,7 +3001,7 @@ static void readMeasure(Measure* m, int staffIdx, XmlReader& e)
                   clef->setTrack(e.track());
                   clef->read(e);
                   clef->setGenerated(false);
-                  if (e.tick().isZero()) {
+                  if (e.tick().isZero() && clef->segment()) { // if segment(), then it's a header clef
```

Comment:
But `clef` hasn't been yet added anywhere, wouldn't this condition be always false?

Reply:
Ah, it's true. But my test shows sometimes this works :O Don't know why.

Reply:
So, um, what's the exact way to judge if a clef is header clef?

Reply:
If it's already added to the score, then you check the segment type (CLEF vs HEADER_CLEF).  But if it's not added to the score, this is probably the wrong place to be making decisions on that basis.

Reply:
Or is there a way to check if at the same tick, the clef is placed before the time signature? That can do the same thing.

Reply:
Ah, we can simply check if the clef segment is the first one in the measure. See the new commit message.